src/main/java/io/gatling/mojo/EnterpriseUploadMojo.java

```diff
@@ -51,6 +54,21 @@ public final class EnterpriseUploadMojo extends AbstractEnterprisePluginMojo {
 
   @Override
   public void execute() throws MojoFailureException {
+    final BatchEnterprisePlugin enterprisePlugin = initBatchEnterprisePlugin();
+
+    final String projectDirectory = mavenProject.getBasedir().getAbsolutePath();
+    final String jsonConfig = PackageConfiguration.loadToJson(projectDirectory);
+
+    if (StringUtils.isNotBlank(jsonConfig)) {
```

Comment:
This clean up should be done in `PackageConfiguration.loadToJson`. Client code should only have to test `null`.

Reply:
https://github.com/gatling/gatling-enterprise-plugin-commons/pull/114

Reply:
Please upgrade to commons 1.9.0-M7 so you can only test against null.

Reply:
Done